src/server/node_services/nodes_client.js

```diff
@@ -324,19 +324,19 @@ class NodesClient {
 
         const { hosts } = await this.list_hosts_by_pool(pool_name, system_id, auth_token);
         const promise_list = hosts
-            .sort((host1, host2) => {
-                const host1_seq = Number(host1.name.match(/-(\d+)#/)[1]);
-                const host2_seq = Number(host2.name.match(/-(\d+)#/)[1]);
-                return host1_seq - host2_seq;
-            })
+            // .sort((host1, host2) => {
```

Comment:
any reason to remove this? if yes then just delete it

Reply:
yes. we changed the naming convention... we don't have order any more... so sorting isn't helpful and not working anymore as well (the last part is just random string that can contain not just numbers)

Reply:
cleaned it up